tsdb/engine/tsm1/engine.go

```diff
@@ -1766,19 +1770,20 @@ func (e *Engine) deleteSeriesRange(seriesKeys [][]byte, min, max int64) error {
 			return err
 		}
 
-		fielsetChanged := false
+		actuallyDeleted := make([]string, 0, len(measurements))
 		for k := range measurements {
 			if dropped, err := e.index.DropMeasurementIfSeriesNotExist([]byte(k)); err != nil {
 				return err
 			} else if dropped {
-				if err := e.cleanupMeasurement([]byte(k)); err != nil {
+				if deleted, err := e.cleanupMeasurement([]byte(k)); err != nil {
 					return err
```

Comment:
If we return here because `cleanupMeasurement` fails, do we need to do anything if `actuallyDeleted` is not empty?

Reply:
I think we want to fail the whole call, but I am open to other ideas.....

Reply:
That seems reasonable. I was more concerned with making sure this had been thought about.